account_invoice_auto_send_mail/README.rst

```diff
@@ -0,0 +1 @@
+Allows you to automate the sending of customer sales invoices and rectifiers to send by email
```

Comment:
You actually don't need README.rst, just make sure the content in "readme" folder is right. When this PR is merged, README.rst will auto generated for you.
A good example of "readme" https://github.com/OCA/mis-builder/tree/12.0/mis_builder/readme

Reply:
Ok i delete the file